.github/advanced-issue-labeler.yml

```diff
@@ -0,0 +1,76 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+# syntax - https://github.com/redhat-plumbers-in-action/advanced-issue-labeler#policy
+# Below keys map from the option used in issue form dropdowns to issue labels
+# Limitation:
+# Currently it's not possible to use strings containing ,␣ in the dropdown menus in the issue forms.
+
+---
+
+policy:
+  - template: [bug.yml, feature.yml]
+    section:
+      - id: [frequency]
+        label:
+          - name: 'Frequency:Once'
+            keys: ['Once']
+          - name: 'Frequency:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Frequency:Always'
+            keys: ['Always']
+
+      - id: [repro]
+        label:
+          - name: 'Repro:Always'
+            keys: ['Always']
+          - name: 'Repro:Sometimes'
+            keys: ['Sometimes']
+          - name: 'Repro:No'
+            keys: ['No']
+
+      - id: [sync, flavour, services]
+        block-list: []
+        label:
+          - name: 'SDK-Use:Local'
+            keys: ['Local Database only']
+          - name: 'SDK-Use:Sync'
+            keys: ['Atlas Device Sync']
+          - name: 'SDK-Use:Services'
+            keys: ['Atlas App Services: Function or GraphQL or DataAPI etc']
+          - name: ['SDK-Use:All']
+            keys: ['Both Atlas Device Sync and Atlas App Services']
+
+      - id: [encryption]
+        block-list: []
+        label:
+          - name: 'Encryption:On'
+            keys: ['Yes']
+          - name: 'Encryption:Off'
+            keys: ['No']
+
+      - id: [app-type]
+        block-list: []
+        label:
+          - name: 'App-type:Unity'
```

Comment:
These all look C# related?

Reply:
Yeah - unfortunately, this file is shared across all repos, so this needs to cover all possible values for `app-type`. Right now this field is not used in the C++ SDK, but if we want to label different app types developed with the c++ SDK, let me know what those are and I'll add them to the list.

Reply:
I guess we capture OS and version which is enough for now. But we might want to break this down further in the future for platform integrations such as Qt etc. Happy to resolve.